content/pages/features.liquid

```diff
@@ -0,0 +1,136 @@
+---
+permalink: /pages/features/
+layout: layouts/page
+
+title: cloud.gov Pages - Features
+hidetitle: true
+---
+
```

Comment:
wait this one is liquid -- could contact match?

Reply:
Made contact an html extension since there doesn't appear to be any liquid code there.